doc/extensions/only/eager_only.py

```diff
@@ -44,3 +44,13 @@ def run(self, *args):
 
 def setup(app):
     directives.register_directive('only', EagerOnly)
+
+    # The tags.add call above is setting tags.tags['TRUE'] = True.
+    # The operation is idempotent and will have taken effect before
+    # the next eval_condition() which may rely on it so this is thread
+    # safe both for read and writes (all other operations are local to
+    # the local nodes variable).
+    return {
+        'parallel_read_safe': True,
```

Comment:
Thanks for evaluating these Sphinx extensions.  We're not running Sphinx with the ``-j`` option (yet) because we noticed some differences in the output and error messages, but marking these extensions as thread-safe will help the cause as we evaluate new versions of Sphinx/Breathe...